<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset='utf-8'>
  <meta name="viewport" content="width=device-width, initial-scale=1" id="wixDesktopViewport" />
  <title>Publications | Zeng Lab</title>
  <link rel="shortcut icon" href="favicon.ico">
  <link rel="stylesheet" type="text/css" href="css/style.css?t=2"/>
  <link rel="stylesheet" type="text/css" href="css/iconfont.css?t=1"/>
</head>
<body>
<header>
	<a href="/" ><img id="logo" class="logo" src="img/logo.png" title="西湖大学" /></a>
	<div class="title">
		<h1>Zeng Research Group</h1>
		<h2>Artificial Intelligence and Computational Biology  <br />  Laboratory</h2>
	</div>
</header>
<ul class="menu">
	<li><a href="index.html">Home</a></li>
	<li><a href="research.html">Research</a></li>
	<li><a href="people.html">People</a></li>
	<li class="active"><a href="publications.html">Publications</a></li>
	<li><a href="openings.html">Openings</a></li>
</ul>

<div class="publicationsList">
	<p class="warning">
    Papers (*: Corresponding author; #: Co-first author)
</p>
<div class="container">
    <ol reversed="" class="content">
        <li>
            <p>
                Peizhuo Wang, Xiao Wen, Han Li, Peng Lang, Shuya Li, Yipin Lei, Hantao Shu, Lin Gao, Dan Zhao*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>A network-based framework for deciphering driver regulators of cell fate decisions from single-cell RNA-seq data.</strong>
            </p>
            <p>
                <em>Nature Communications 2023.</em>
            </p>
        </li>
        <li>
            <p>
                Han Li, Ruotian Zhang, Yaosen Min, Dacheng Ma, Dan Zhao*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>A Knowledge-Guided Pre-training Framework for Improving Molecular Representation Learning.</strong>
            </p>
            <p>
                <em>Nature Communications 2023.</em>
            </p>
        </li>
        <li>
            <p>
                Tingzhong Tian, Shuya Li, Meng Fang, Dan Zhao*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>MolSHAP: Interpreting Quantitative Structure–Activity Relationships Using Shapley Values of R-Groups.</strong>
            </p>
            <p>
                <em>Journal of Chemical Information and Modeling. 2023.</em>
            </p>
        </li>
        <li>
            <p>
                Han Li, Xuan He, Lawrence Kurowski, Ruotian Zhang Dan Zhao*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>Improving comparative analyses of Hi-C data via contrastive self-supervised learning.</strong>
            </p>
            <p>
                <em>Briefings in Bioinformatics. 2023.</em>
            </p>
        </li>
        <li>
            <p>
                Xingang Peng#, Yipin Lei#, Peiyuan Feng, Lemei Jia, Jianzhu Ma, Dan Zhao*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>Characterizing the interaction conformation between T cell receptors and epitopes with deep learning.</strong>
            </p>
            <p>
                <em>Nature Machine Intelligence. 2023.</em>
            </p>
        </li>
        <li>
            <p>
                Shuya Li#, Tingzhong Tian#, Ziting Zhang, Ziheng Zou, Dan Zhao* and Jianyang Zeng*.
            </p>
            <p>
                <strong>PocketAnchor: Learning Structure-based Pocket Representations for Protein-Ligand Interaction Prediction.</strong>
            </p>
            <p>
                <em>Cell Systems. 2023. <em>Cover Article.</em></em>
            </p>
        </li>
        <li>
            <p>
                Chang Liu#, Cuinan Yu#, Yipin Lei#, Kangbo Lyu, Tingzhong Tian, Qianhao Li, Dan Zhao*, Fengfeng Zhou*, Jianyang Zeng*.
            </p>
            <p>
                <strong>Improving target-disease association prediction through a graph neural network with credibility information.</strong>
            </p>
            <p>
                <em>Pacific Symposium on Biocomputing. 2023.</em>
            </p>
        </li>
        <li>
            <p>
                Han Li, Dan Zhao*, Jianyang Zeng*.
            </p>
            <p>
                <strong>KPGT: knowledge-guided pre-training of graph transformer for molecular property prediction. </strong>
            </p>
            <p>
                <em>Proceedings of the 28th ACM SIGKDD Conference on Knowledge Discovery and Data Mining (KDD), 2022, 857-867.</em>
            </p>
        </li>
        <li>
            <p>
                Siqin Zhang, Kuo Yang, Zhenhong Liu, Xinxing Lai, Zhen Yang, Jianyang Zeng, Shao Li*.
            </p>
            <p>
                <strong>DrugAI: a multi-view deep learning model for predicting drug–target activating/inhibiting mechanisms.</strong>
            </p>
            <p>
                <em>Briefings in Bioinformatics. 2022.</em>
            </p>
        </li>
        <li>
            <p>
                Han Li, Xinyi Zhao, Shuya Li, Fangping Wan, Dan Zhao*, Jianyang Zeng*.
            </p>
            <p>
                <strong>Improving molecular property prediction through a task similarity enhanced transfer learning strategy.</strong>
            </p>
            <p>
                <em>iScience. 2022.</em>
            </p>
        </li>
        <li>
            <p>
                Hantao Shu, Fan Ding, Jingtian Zhou, Yexiang Xue, Dan Zhao, Jianyang Zeng*, Jianzhu Ma*.
            </p>
            <p>
                <strong>Boosting single-cell gene regulatory network reconstruction via bulk-cell transcriptomic data.</strong>
            </p>
            <p>
                <em>Briefings in Bioinformatics. 2022.</em>
            </p>
        </li>
        <li>
            <p>
                Xiaolong Feng#, Xiaofan Zhang#, Shuangying Jiang, Yuanwei Tang, Chao Cheng, Parthasarathy Abinand Krishna, Xiaoting Wang, Junbiao Dai, Jianyang Zeng*, Tian Xia*, Dan Zhao*.
            </p>
            <p>
                <strong>A DNA-based non-infectious replicon system to study SARS-CoV-2 RNA synthesis.</strong> 
            </p>
            <p>
                <em>Computational and Structural Biotechnology Journal. 2022.</em>
            </p>
        </li>
        <li>
            <p>
                Eugene F Douglass Jr.#, Robert J Allaway#, Bence Szalai#, Wenyu Wang, Tingzhong Tian, Adrià Fernández-Torras, Ron Realubit, Charles Karan, Shuyu Zheng, Alberto Pessia, Ziaurrehman Tanoli, Mohieddin Jafari, Fangping Wan, Shuya Li, Yuanpeng Xiong, Miquel Duran-Frigola, Martino Bertoni, Pau Badia-i-Mompel, Lídia Mateo, Oriol Guitart-Pla, Verena Chung, Jing Tang, Jianyang Zeng, Patrick Aloy, Julio Saez- Rodriguez*, Justin Guinney*, Daniela S Gerhard*, Andrea Califano*, Challenge Consortium.
            </p>
            <p>
                <strong>A Community Challenge for Pancancer Drug Mechanism of Action Inference from Perturbational Profile Data.</strong> 
            </p>
            <p>
                <em>Cell Reports Medicine. 2022.</em>
            </p>
        </li>
        <li>
            <p>
                Wuai Zhou, Kuo Yang, Jianyang Zeng, Xinxing Lai, Xin Wang, Chaofan Ji, Yan Li, Peng Zhang, Shao Li*.
            </p>
            <p>
                <strong>FordNet: recommending traditional Chinese medicine formula via deep neural network integrating phenotype and molecule.</strong> 
            </p>
            <p>
                <em>Pharmacological Research. 2021.</em>
            </p>
        </li>
        <li>
            <p>
                Yipin Lei, Shuya Li, Ziyi Liu, Fangping Wan, Tingzhong Tian, Shao Li, Dan Zhao*, Jianyang Zeng*.
            </p>
            <p>
                <strong>A deep-learning framework for multi-level peptide–protein interaction prediction.</strong> 
            </p>
            <p>
                <em>Nature Communications. 2021.</em>
            </p>
        </li>
        <li>
            <p>
                Zhaofeng Ye, Fengling Chen, Jiangyang Zeng, Juntao Gao*, Michael Q. Zhang*.
            </p>
            <p>
                <strong>ScaffComb: A Phenotype-Based Framework for Drug Combination Virtual Screening in Large-Scale Chemical Datasets.</strong>
            </p>
            <p>
                <em>Advanced Science. 2021.</em>
            </p>
        </li>
        <li>
            <p>
                Hantao Shu, Jingtian Zhou, Qiuyu Lian, Han Li, Dan Zhao, Jianyang Zeng*, Jianzhu Ma*.
            </p>
            <p>
                <strong>Modeling gene regulatory networks using neural network architectures.</strong> 
            </p>
            <p>
                <em>Nature Computational Science. 2021.</em>
            </p>
        </li>
        <li>
            <p>
                Peiyuan Feng, Jianyang Zeng*, Jianzhu Ma*.
            </p>
            <p>
                <strong>Predicting MHC-peptide binding affinity by differential boundary tree.</strong> 
            </p>
            <p>
                <em>ISMB 2021.<br/> Bioinformatics. 2021.</em>
            </p>
        </li>
        <li>
            <p>
                Anna Cichonska, Balaguru Ravikumar, Robert J Allaway, Sungjoon Park, Fangping Wan, Olexandr Isayev, Shuya Li, Michael J Mason, Andrew Lamb, Minji Jeon, Sunkyu Kim, Mariya Popova, Jianyang Zeng, Kristen Dang, Gregory Koytiger, Jaewoo Kang, Carrow I Wells, Timothy M Willson, Tudor I Oprea, Avner Schlessinger, David H Drewry, Gustavo A Stolovitzky, Krister Wennerberg, Justin Guinney*, Tero Aittokallio*.
            </p>
            <p>
                <strong>Crowdsourced mapping of unexplored target space of kinase inhibitors.</strong> 
            </p>
            <p>
                <em>Nature Communications. 2021.</em>
            </p>
        </li>
        <li>
            <p>
                Peiyuan Feng#, An Xiao#, Meng Fang, Fangping Wan, Shuya Li, Peng Lang, Dan Zhao*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>A novel machine learning based framework for modeling transcription elongation.</strong> 
            </p>
            <p>
                <em>Proceedings of the National Academy of Sciences (PNAS). 2021.</em>
            </p>
        </li>
        <li>
            <p>
                Tingzhong Tian, Shuya Li, Peng Lang, Dan Zhao*, Jianyang Zeng*.
            </p>
            <p>
                <strong>Full-length ribosome density prediction by a multi-input and multi-output model.</strong> 
            </p>
            <p>
                <em>PLOS Computational Biology. 2021.</em>
            </p>
        </li>
        <li>
            <p>
                Dan Zhao#, Weifan Xu#, Xiaofan Zhang#, Xiaoting Wang, Enming Yuan, Yuanpeng Xiong, Shenyang Wu, Shuya Li, Nian Wu, Tingzhong Tian, Xiaolong Feng, Hantao Shu, Peng Lang, Xiaokun Shen, Haitao Li, Pilong Li,* and Jianyang Zeng*.
            </p>
            <p>
                <strong>Understanding the phase separation characteristics of nucleocapsid protein provides a new therapeutic opportunity against SARS-CoV-2.</strong> 
            </p>
            <p>
                <em>Protein &amp; Cell. 2021. </em>
            </p>
        </li>
        <li>
            <p>
                Yuanpeng Xiong#, Xuan He#, Dan Zhao, Tingzhong Tian, Lixiang Hong, Tao Jiang*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>Modelling Multi-species RNA Modification through Multi-task Curriculum Learning.</strong> 
            </p>
            <p>
                <em>Nucleic Acids Research. 2021.</em>
            </p>
        </li>
        <li>
            <p>
                Yiyue Ge#, Tingzhong Tian#, Suling Huang#, Fangping Wan#, Jingxin Li#, Shuya Li, Xiaoting Wang, Hui Yang, Lixiang Hong, Nian Wu, Enming Yuan, Yunan Luo, Lili Cheng, Chengliang Hu, Yipin Lei, Hantao Shu, Xiaolong Feng, Ziyuan Jiang, Yunfu Wu, Ying Chi, Xiling Guo, Lunbiao Cui, Liang Xiao, Zeng Li, Chunhao Yang, Zehong Miao, Ligong Chen, Haitao Li, Hainian Zeng, Dan Zhao*, Fengcai Zhu*, Xiaokun Shen*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>An integrative drug repositioning framework discovered a potential therapeutic agent targeting COVID-19.</strong> 
            </p>
            <p>
                <em>Signal Transduction and Targeted Therapy. 2021. </em>
            </p>
        </li>
        <li>
            <p>
                Hailin Hu# , Xianggen Liu#; , An Xiao, YangYang Li, Chengdong Zhang, Tao Jiang, Dan Zhao, Sen Song*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>Riboexp: an interpretable reinforcement learning framework for ribosome density modeling.</strong> 
            </p>
            <p>
                <em>Briefings in Bioinformatics. 2021. </em>
            </p>
        </li>
        <li>
            <p>
                Han Li, Xinyi Zhao, Shuya Li, Fangping Wan, Dan Zhao*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>MoTSE: an interpretable task similarity estimator for small molecular property prediction tasks.</strong> 
            </p>
            <p>
                <em>RECOMB. 2021.</em>
            </p>
        </li>
        <li>
            <p>
                Lixiang Hong, Jinjian Lin, Shuya Li, Fangping Wan, Hui Yang, Tao Jiang, Dan Zhao*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>A novel machine learning framework for automated biomedical relation extraction from large-scale literature repositories.</strong> 
            </p>
            <p>
                <em>Nature Machine Intelligence. 2020.</em>
            </p>
        </li>
        <li>
            <p>
                Shuya Li#, Fangping Wan#, Hantao Shu, Tao Jiang, Dan Zhao*, Jianyang Zeng*.
            </p>
            <p>
                <strong>MONN: A Multi-objective Neural Network for Predicting Compound-Protein Interactions and Affinities.</strong> 
            </p>
            <p>
                <em>Cell Systems. 2020.</em>
            </p>
        </li>
        <li>
            <p>
                Fangping Wan#, Shuya Li#, Tingzhong Tian, Yipin Lei, Dan Zhao*, Jianyang Zeng*.
            </p>
            <p>
                <strong>EXP2SL: a Machine Learning Framework for Cell-Line Specific Synthetic Lethality Prediction.</strong> 
            </p>
            <p>
                <em>Frontiers in Pharmacology. 2020.</em>
            </p>
        </li>
        <li>
            <p>
                Rong Ma#, Yi Li#, Chenxing Li#, Fangping Wan, Hailin Hu, Wei Xu*, Jianyang Zeng*.
            </p>
            <p>
                <strong>Secure multiparty computation for privacy-preserving drug discovery.</strong> 
            </p>
            <p>
                <em>Bioinformatics. 2020.</em>
            </p>
        </li>
        <li>
            <p>
                Shuya Li#, FangpingWan#, Hantao Shu, Tao Jiang, Dan Zhao*, and Jianyang Zeng*.
            </p>
            <p>
                <strong>MONN: a Multi-Objective Neural Network for Predicting Pairwise Non-Covalent Interactions and Binding Affinities between Compounds and Proteins.</strong> 
            </p>
            <p>
                RE<em>COMB 2020.<br/>Also invited to Cell Systems. 2020.</em>
            </p>
        </li>
        <li>
            <p>
                Hongfei Cui, Hailin Hu, Jianyang Zeng*, Ting Chen*.
            </p>
            <p>
                <strong>DeepShape: estimating isoform-level ribosome abundance and distribution with Ribo-seq data.</strong> 
            </p>
            <p>
                <em>ICIBM 2019. <em>Best Paper Award.</em><br/> BMC Bioinformatics. 2019.</em>
            </p>
        </li>
        <li>
            <p>
                Yan Hu, Ziqiang Wang, Hailin Hu, Fangping Wan, Lin Chen, Yuanpeng Xiong, Xiaoxia Wang, Dan Zhao, Weiren Huang*, Jianyang Zeng*.
            </p>
            <p>
                <strong>ACME: pan-specific peptide-MHC class I binding prediction through attention-based deep neural networks.</strong>
            </p>
            <p>
                <em>Bioinformatics. 2019.</em>
            </p>
        </li>
        <li>
            <p>
                Hao Chen, Dipan Shaw, Jianyang Zeng, Dongbo Bu, Tao Jiang*.
            </p>
            <p>
                <strong>DIFFUSE: predicting isoform functions from sequences and expression profiles via deep learning.</strong> 
            </p>
            <p>
                <em>ISMB. Bioinformatics. 2019.</em>
            </p>
        </li>
        <li>
            <p>
                Fangping Wan, Yue Zhu, Hailin Hu, Antao Dai, Xiaoqing Cai, Ligong Chen, Haipeng Gong, Tian Xia, Dehua Yang*, Mingwei Wang*, Jianyang Zeng*.
            </p>
            <p>
                <strong>A deep learning based framework for large-scale in silico drug screening.</strong> 
            </p>
            <p>
                <em>Genomics, Proteomics &amp; Bioinformatics. 2019. </em>
            </p>
        </li>
        <li>
            <p>
                Xuan He, Sai Zhang, Yanqing Zhang, Zhixin Lei, Tao Jiang, Jianyang Zeng*.
            </p>
            <p>
                <strong>Characterizing RNA pseudouridylation by convolutional neural networks.</strong> 
            </p>
            <p>
                <em>Genomics, Proteomics &amp; Bioinformatics. 2019. </em>
            </p>
        </li>
        <li>
            <p>
                Ahmed Abbas, Xuan He, Jing Niu, Bin Zhou, Guangxiang Zhu, Tszshan Ma, Jiangpeikun Song, Juntao Gao, Michael Q Zhang, Jianyang Zeng*.
            </p>
            <p>
                <strong>Integrating Hi-C and FISH data for modeling of the 3D organization of chromosomes.</strong> 
            </p>
            <p>
                <em>Nature Communications. 2019.</em>
            </p>
        </li>
        <li>
            <p>
                Yunan Luo, Yun William Yu, Jianyang Zeng, Bonnie Berger*, Jian Peng*.
            </p>
            <p>
                <strong>Metagenomic binning through low-density hashing.</strong> 
            </p>
            <p>
                <em>Bioinformatics. 2018.</em>
            </p>
        </li>
        <li>
            <p>
                Fangping Wan, Lixiang Hong, An Xiao, Tao Jiang, Jianyang Zeng*.
            </p>
            <p>
                <strong>NeoDTI: Neural integration of neighbor information from a heterogeneous network for discovering new drug-target interactions.</strong> 
            </p>
            <p>
                <em>Bioinformatics. 2018.</em>
            </p>
        </li>
        <li>
            <p>
                Hailin Hu, An Xiao, Sai Zhang, Yangyang Li, Xuanling Shi, Tao Jiang, Linqi Zhang, Lei Zhang*, Jianyang Zeng*.
            </p>
            <p>
                <strong>DeepHINT: Understanding HIV-1 integration via deep learning with attention.</strong> 
            </p>
            <p>
                <em>Bioinformatics. 2018.</em>
            </p>
        </li>
        <li>
            <p>
                Guangxiang Zhu, Wenxuan Deng, Hailin Hu, Rui Ma, Sai Zhang, Jinglin Yang, Jian Peng, Tommy Kaplan, Jianyang Zeng*.
            </p>
            <p>
                <strong>Reconstructing spatial organizations of chromosomes through manifold learning.</strong> 
            </p>
            <p>
                <em>Nucleic Acids Research. 2018.</em>
            </p>
        </li>
        <li>
            <p>
                Xiaoliang Ren; Runsheng Li; Xiaolin Wei, Yu Bi, Wing Ho, Qiutao Ding, Zhihong Zhong, Amanda Young, Chia-Ling Hsieh, Jianyang Zeng*, Xiao Liu*, Zhongying Zhao*.
            </p>
            <p>
                <strong>Genomic bases of recombination suppression in the hybrid between hermaphroditic Caenorhabditis briggsae and gonochoristic C. nigoni.</strong> 
            </p>
            <p>
                <em>Nucleic Acids Research. 2018.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyu Zhou, Shining Ma, Dongfang Wang, Jianyang Zeng, Tao Jiang*.
            </p>
            <p>
                <strong>FreePSI: an alignment-free approach to estimating exon-inclusion ratios without a reference transcriptome.</strong> 
            </p>
            <p>
                <em>Nucleic Acids Research. 2017.</em>
            </p>
        </li>
        <li>
            <p>
                Zhenhai Du, Hui Zheng, Bo Huang, Rui Ma, Jingyi Wu, Xianglin Zhang, Jing He, Yunlong Xiang, Qiujun Wang, Yuanyuan Li, Jing Ma, Xu Zhang, Ke Zhang, Yang Wang, Michael Q. Zhang, Juntao Gao, Jesse R. Dixon, Xiaowo Wang, Jianyang Zeng and Wei Xie*.
            </p>
            <p>
                <strong>Allelic reprogramming of 3D chromatin architecture during early mammalian development.</strong> 
            </p>
            <p>
                <em>Nature. 2017.</em>
            </p>
        </li>
        <li>
            <p>
                Yunan Luo#, Xinbin Zhao#, Jingtian Zhou#, Jinling Yang, Yanqing Zhang, Wenhua Kuang, Jian Peng*, Ligong Chen* and Jianyang Zeng*.
            </p>
            <p>
                <strong>A Network Integration Approach for Drug-Target Interaction Prediction and Computational Drug Repositioning from Heterogeneous Information.</strong> 
            </p>
            <p>
                Na<em>ture Communications. 2017.</em>
            </p>
        </li>
        <li>
            <p>
                Shuya Li#, Fanghong Dong#, Yuexin Wu#, Sai Zhang, Chen Zhang, Xiao Liu, Tao Jiang, Jianyang Zeng*.
            </p>
            <p>
                <strong>A deep boosting based approach for capturing the sequence binding preferences of RNA-binding proteins from high-throughput CLIP-seq data.</strong> 
            </p>
            <p>
                <em> Nucleic Acids Research. 2017.</em>
            </p>
        </li>
        <li>
            <p>
                Sai Zhang#, Hailin Hu#, Jingtian Zhou#, Xuan He, Tao Jiang, Jianyang Zeng*.
            </p>
            <p>
                <strong>Analysis of ribosome stalling and translation elongation dynamics by deep learning.</strong> 
            </p>
            <p>
                <em>Cell Systems. 2017. In press.</em>
            </p>
        </li>
        <li>
            <p>
                Dapeng Xiong, Jianyang Zeng*, Haipeng Gong*.
            </p>
            <p>
                <strong>A deep learning framework for improving long-range residue-residue contact prediction using a hierarchical strategy.</strong>
            </p>
            <p>
                <em>Bioinformatics. 2017.</em>
            </p>
        </li>
        <li>
            <p>
                Sai Zhang, Hailin Hu, Tao Jiang, Lei Zhang* and Jianyang Zeng*.
            </p>
            <p>
                <strong>Predicting translation initiation sites by deep learning.</strong> 
            </p>
            <p>
                <em>ISMB/ECCB. Bioinformatics. 2017.</em>
            </p>
        </li>
        <li>
            <p>
                Yunan Luo, Xinbin Zhao, Jingtian Zhou, Jinling Yang, Yanqing Zhang, Wenhua Kuang, Jian Peng*, Ligong Chen*, Jianyang Zeng*.
            </p>
            <p>
                <strong>A Network Integration Approach for Drug-Target Interaction Prediction and Computational Drug Repositioning from Heterogeneous Information.</strong> 
            </p>
            <p>
                <em> RECOMB. 2017.</em>
            </p>
        </li>
        <li>
            <p>
                Sai Zhang, Hailin Hu, Jingtian Zhou, Xuan He, Tao Jiang and Jianyang Zeng*.
            </p>
            <p>
                <strong>ROSE: a deep learning based framework for predicting ribosome stalling.</strong> 
            </p>
            <p>
                <em>RECOMB. 2017.</em>
            </p>
        </li>
        <li>
            <p>
                Sai Zhang, Muxuan Liang, Zhongjun Zhou, Chen Zhang, Ning Chen, Ting Chen and Jianyang Zeng*.
            </p>
            <p>
                <strong>Elastic Restricted Boltzmann Machines for Cancer Data Analysis.</strong> 
            </p>
            <p>
                <em>Quantitative biology. 2017. <em>Cover Article.</em></em>
            </p>
        </li>
        <li>
            <p>
                Feng Wang, Huichao Gong, Gaochao Liu, Meijing Lia, Chuangye Yana, Tian Xia*, Xueming Li*, Jianyang Zeng*.
            </p>
            <p>
                <strong>DeepPicker: A deep learning approach for fully automated particle picking in cryo-EM.</strong> 
            </p>
            <p>
                <em>Journal of Structural Biology. Volume 195, Issue 3. 2016.</em>
            </p>
        </li>
        <li>
            <p>
                Yuchao Pan, Yuxi Dong, Jingtian Zhou, Mark Hallen, Bruce R. Donald*, Jianyang Zeng* and Wei Xu*.
            </p>
            <p>
                <strong>cOSPREY: A Cloud-Based Distributed Algorithm for Large-Scale Computational Protein Design.</strong> 
            </p>
            <p>
                <em>Journal of Computational Biology. 2016. <a href="https://github.com/iiisthu/cOSPREY">Code</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Yunan Luo, Jianyang Zeng, Bonnie Berger* and Jian Peng*.
            </p>
            <p>
                <strong>Low-density localitysensitive hashing boosts metagenomic binning.</strong> 
            </p>
            <p>
                <em>International Conference on Research in Computational Molecular Biology (RECOMB). 2016.</em>
            </p>
        </li>
        <li>
            <p>
                Sai Zhang, Jingtian Zhou, Hailin Hu, Haipeng Gong, Ligong Chen, Chao Cheng*, Jianyang Zeng*.
            </p>
            <p>
                <strong>A deep learning framework for modeling structural features of RNA-binding protein targets.</strong> 
            </p>
            <p>
                <em>Nucleic Acids Research. 2015. </em>
            </p>
        </li>
        <li>
            <p>
                Yichao Zhou, Bruce R. Donald, Jianyang Zeng*.
            </p>
            <p>
                <strong>Parallel computational protein design.</strong> 
            </p>
            <p>
                <em>Methods in Molecular Biology. 2015.</em>
            </p>
        </li>
        <li>
            <p>
                Dapeng Xiong, Jianyang Zeng and Haipeng Gong*.
            </p>
            <p>
                <strong>RBRIdent: an algorithm for improved identification of RNA-binding residues in proteins from primary sequences.</strong>
            </p>
            <p>
                <em>Proteins: Structure, Function, and Bioinformatics. 2015.</em>
            </p>
        </li>
        <li>
            <p>
                Siyu Wang, Jinbo Xu and Jianyang Zeng*.
            </p>
            <p>
                <strong>Inferential modeling of 3D chromatin structure.</strong> 
            </p>
            <p>
                <em>Nucleic Acids Research. 2015. </em>
            </p>
        </li>
        <li>
            <p>
                Huichao Gong, Sai Zhang, Jiangdian Wang, Haipeng Gong and Jianyang Zeng*.
            </p>
            <p>
                <strong>Constructing Structure Ensembles of Intrinsically Disordered Proteins from Chemical Shift Data.</strong> 
            </p>
            <p>
                <em>International Conference on Research in Computational Molecular Biology (<strong>RECOMB</strong> 2015).<br/> Also in the Journal of Computational Biology. 2015.</em>
            </p>
        </li>
        <li>
            <p>
                Yichao Zhou, Yuexin Wu and Jianyang Zeng*.
            </p>
            <p>
                <strong>Computational Protein Design Using AND/OR Branch-and-Bound Search.</strong> 
            </p>
            <p>
                <em>International Conference on Research in Computational Molecular Biology (<strong>RECOMB</strong> 2015).<br/> Also in the Journal of Computational Biology. 2015.</em>
            </p>
        </li>
        <li>
            <p>
                Muxuan Liang, Zhizhong Li, Ting Chen and Jianyang Zeng*.
            </p>
            <p>
                <strong>Integrative Data Analysis of Multi-platform Cancer Data with a Multimodal Deep Learning Approach.</strong> <em>IEEE/ACM Transactions on Computational Biology and Bioinformatics. 2015. <a href="https://github.com/muxuanliang/multideep">Code</a>.</em>
            </p>
        </li>
        <li>
            Yichao Zhou and Jianyang Zeng*. <strong>Massively Parallel A* Search on a GPU.</strong> <em>Proceeding of the Twenty-Ninth AAAI Conference on Artificial Intelligence (<strong>AAAI 2015</strong>). 2015. Oral Paper. <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/aaai_2015.pdf">PDF</a>, <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/aaai2015apx.pdf">Appendix</a>. <a href="https://github.com/zhou13/uastar">Code</a>. <br/>[Acceptance Rate (Oral): 11.95%]</em>
        </li>
        <li>
            Yufeng Liu, Jianyang Zeng and Haipeng Gong*. <strong>Improving the orientation-dependent statistical potential using a reference state.</strong> <em>Proteins: Structure, Function, and Bioinformatics. 82(10):2383-93. 2014. <a href="https://onlinelibrary-wiley-com-443.webvpn.westlake.edu.cn/doi/abs/10.1002/prot.24600">PDF</a>.</em>
        </li>
        <li>
            Yichao Zhou, Wei Xu, Bruce R. Donald, Jianyang Zeng*. <strong>An Efficient Parallel Algorithm for Accelerating Computational Protein Design.</strong> <em>ISMB. July 2014.<br/> Bioinformatics. <a href="https://watermark-silverchair-com-s.webvpn.westlake.edu.cn/bioinformatics_30_12_i255.pdf?token=AQECAHi208BE49Ooan9kkhW_Ercy7Dm3ZL_9Cf3qfKAc485ysgAAA4AwggN8BgkqhkiG9w0BBwagggNtMIIDaQIBADCCA2IGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMh_9BobfgqleRhlJTAgEQgIIDM2OmXXWgjYSmCoCrV9BkztCJ_5r38KNUTzyK0QJORtZWD2H6Othts6SbY1FRzCAWbOk-JhDLU5b3-EcgtKf9JJCyLA32et0506INBxGucP7RLJagV-vfRiUzbUUiIwREiadGJtmWn3awI5mBVs4YvHMRkHTRylb2vgB0vsZ_vXfNak76C21cC8sDkF5mQdsFO0BhQ0yxVXwnAkGmWWcj4qGKw5Wn0UzlFlb9sKB34LmTs55rUs6HSYTDs1SoA4cZw6_XDxSAn4VI4fP6T-cMMXn2WsdzMwvrBnqIJHBp-czF8y9z3AEwuQaeJyc-OAu1TJHkRZIrfsSM674hypiLbeXjCEkLdU70SbdGk_21RdfhcJ7OUdl6X8PeaEKyAaypvtk5CY49ZgK11oEJdn78BFN9uV5LPqIYLCX7iTQZlH9gq4NcB-Hp-KbNaZ5QjZ_KUla1PMrYQNZnv4ao7fSUNc5NdE7-Mz-6v9K1M9K2q9pJXaExjVL7-qymBDikXuR6re03500CUbnbg5iU7bTY8y2l0w5XMwcQHHFZwcWu8LLdyZdRWrSS3ZnYWssHE-cV-JLQpE5giAZ3q51VlQIbpnCEor6z5BVTPqZTpq5OUhwbE4GT-4nqsdZ0Wl-QGE1jb1kICFMaFY-GXG1gESFtrnvIwBShxbxHMq2vADinfcvV_rGiUZj8bOudnuJr-ae42I4LdWG5bAVl0OTn1KWn_unJp3PbkHtiaMVmYFo2aWFj9UXpGFMax3nD0b9frYEtXKhplAtN94DJD0cT6UcTjSNc_cMyLoBG62zUhwBvKF-zT4H8u-Jss4mpO2qVfTcL3WMRQ6aZUlVFEsAwGIyPhnNmvEYdw4tZw4iazPCKz16SzthFNuUymRzaASRSC5tc0BdC_H6nYI2_QUeflUzXDBH8bXaEUrWHupZ3mDUQWr6yOcCCNOJDXjEzAZ4bIzqzsrULc8PZ9Op49thO7QsaikP02gUokNgfXbSvUVv4k9efrgbZKSSxf2JqBLN_3MoON8mEZGTvqXwaDBVZMKF2ybr_MrTAFKJlQNykzAa4YRH9Hy4e3XOqA1Tx2VnKFtAUje3UEA">PDF</a>, <a href="https://zhou13.github.io/gOSPREY/">Software</a>.</em>
        </li>
        <li>
            <p>
                Fan Yang, Jinbo Xu and Jianyang Zeng*.
            </p>
            <p>
                <strong>Drug-Target Interaction Prediction by Integrating Chemical, Genomic, Functional and Pharmacological Data.</strong> 
            </p>
            <p>
                <em>Pacific Symposium on Biocomputing (PSB). 2014. <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/ismb2013.pdf">PDF</a>,<a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/ismb2013_sm.pdf">Supplementary Material</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Yuhao Wang, Jianyang Zeng*.
            </p>
            <p>
                <strong>Predicting Drug-Target Interactions Using Restricted Boltzmann Machines.</strong> 
            </p>
            <p>
                <em> Proceedings of 21st Annual International Conference on Intelligent Systems for Molecular Biology (ISMB)/12th European Conference on Computational Biology (ECCB). July 2013. <br/>Bioinformatics. 2013 July 1; 29(13): i126-i134. <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/ismb2013_sm.pdf">PDF</a>, <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/ismb2013_sm.pdf">Supplementary Material</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng*, Pei Zhou, Bruce R. Donald*.
            </p>
            <p>
                <strong>HASH: a Program to Accurately Predict Protein Ha Shifts from Neighboring Backbone Shifts.</strong> 
            </p>
            <p>
                <em><strong>Journal of Biomolecular NMR.</strong>5(1):105-18, 2013.<a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/jbn13_hash.pdf">PDF</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Chittaranjan Tripathy, Jianyang Zeng, Pei Zhou, Bruce R. Donald*.
            </p>
            <p>
                <strong>Protein Loop Closure using Orientational Restraints from NMR Data.</strong> 
            </p>
            <p>
                <em>Proteins: Structure, Function, and Bioinformatics. 2012. <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/Proteins_08002_c1_sp_Ob.pdf">Cover Article</a>. <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/proteins_2011.pdf">PDF</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Kyle E. Roberts, Pei Zhou, Bruce R. Donald*.
            </p>
            <p>
                <strong>A Bayesian Approach for Determining Protein Side-Chain Rotamer Conformations Using Unassigned NOE Data.</strong> 
            </p>
            <p>
                <em>Journal of Computational Biology. 2011 Nov;18(11):1661-79. Epub 2011 Oct 4. <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/paper_jcb2011.pdf">PDF</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Pei Zhou, Bruce R. Donald*.
            </p>
            <p>
                <strong>Protein Side-Chain Resonance Assignment and NOE Assignment Using RDC-Defined Backbones without TOCSY Data.</strong>
            </p>
            <p>
                <em> Journal of Biomolecular NMR. 2011 Aug;50(4):371-95. Epub 2011 Jun 25. <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/paper_jbnmr2011.pdf">PDF</a>, <a href="https://donaldlab.cs.duke.edu/software/nasca/">Software</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Kyle E. Roberts, Pei Zhou, Bruce R. Donald*.
            </p>
            <p>
                <strong>A Bayesian Approach for Determining Protein Side-Chain Rotamer Conformations Using Unassigned NOE Data.</strong>
            </p>
            <p>
                <em>International Conference on Research in Computational Molecular Biology (<strong>RECOMB&#39;11</strong>). Vancouver, 2011. <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/recomb2011_bayesian.pdf">PDF</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Chittaranjan Tripathy, Jianyang Zeng, Pei Zhou, Bruce R. Donald*.
            </p>
            <p>
                <strong>Protein Loop Closure using Orientational Restraints from NMR Data.</strong> 
            </p>
            <p>
                <em>International Conference on Research in Computational Molecular Biology (<strong>RECOMB&#39;11</strong>). Vancouver, 2011.<a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/recomb2011_pool.pdf">PDF</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Pei Zhou, Bruce R. Donald*.
            </p>
            <p>
                <strong>A Markov Random Field Framework for Protein Side-Chain Resonance Assignment.</strong> <em>I</em>
            </p>
            <p>
                <em>nternational Conference on Research in Computational Molecular Biology (<strong>RECOMB&#39;10</strong>). Lisbon, Portugal. April, 2010. <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/recomb10.pdf">PDF</a>, <a href="https://donaldlab.cs.duke.edu/Supplementary/recomb10/">Supplementary Material</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Jeffrey Boyles, Chittaranjan Tripathy, Lincong Wang, Anthony Yan, Pei Zhou*, Bruce R. Donald*.
            </p>
            <p>
                <strong>High-Resolution Protein Structure Determination Starting with a Global Fold Calculated from Exact Solutions to the RDC Equations.</strong> 
            </p>
            <p>
                <em><strong>Journal of Biomolecular NMR</strong>. 45(3):265-281. 2009. <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/jbn09-epub.pdf">PDF</a>, <a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/supp-matt-jbn09.pdf">Supplementary Material</a>, <a href="https://www.rcsb.org/structure/2KIQ">PDB Structure: 2KIQ</a>. <a href="https://donaldlab.cs.duke.edu/rdcpanda.php">Software</a>. <a href="https://pubmed-ncbi-nlm-nih-gov-443.webvpn.westlake.edu.cn/19711185/">PMID: 19711185</a>.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Chittaranjan Tripathy, Pei Zhou, Bruce R. Donald*.
            </p>
            <p>
                <strong><a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/082Donald.pdf">Hausdorff-Based NOE Assignment Algorithm Using Protein Backbone Determined from Residual Dipolar Couplings and Rotamer Patterns.</a> </strong>
            </p>
            <p>
                <em>Proceedings of the 7th Annual International Conference on Computational Systems Bioinformatics (CSB&#39;08), Stanford CA, pp. 169-181, August 26-29, 2008. <a href="https://donaldlab.cs.duke.edu/Supplementary/csb08/">Supplementary Material</a>. <a href="https://pubmed-ncbi-nlm-nih-gov-443.webvpn.westlake.edu.cn/19642278/">PMID: 19642278</a>. </em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Wen-Jing Hsu.
            </p>
            <p>
                <strong><a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/RAS.pdf">Conflict-Free Container Routing in Mesh Yard Layouts.</a></strong> 
            </p>
            <p>
                <em><strong>Robotics and Autonomous Systems</strong>, Elsevier, Vol. 56(5), pp.451-460, 2008.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Wen-Jing Hsu.
            </p>
            <p>
                <strong><a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/small_world_optimal.pdf">Optimal Routing in a Small-World Network.</a></strong> 
            </p>
            <p>
                <em><strong>Journal of Computer Science and Technology</strong>, 21(4):476-481, 2006.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Wen-Jing Hsu.
            </p>
            <p>
                <strong><a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/small_world_optimal.pdf">Optimal Routing in a Small-World Network.</a></strong> 
            </p>
            <p>
                <em>Proceedings of the 6th International Conference on Parallel andDistributed Computing, Applications and Technologies (PDCAT&#39;05), Dalian, China, 5-8 December 2005. (Acceptancerate: around 25%.) <em>Best Paper Award</em>. </em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng and Wen-Jing Hsu.
            </p>
            <p>
                <strong><a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/ReCord_p2p_network.pdf">ReCord: A Distributed Hash Table with Recursive Structure.</a></strong> 
            </p>
            <p>
                <em>Proceedings of the 6th International Conference on Parallel andDistributed Computing, Applications and Technologies (<strong>PDCAT&#39;05</strong>), Dalian, China, 5-8 December 2005. (Acceptancerate: around 25%.)</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Wen-Jing Hsu and Suiping Zhou.
            </p>
            <p>
                <strong><a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/Internet_evolution.pdf">Construction of Scale-Free Networks with Partial Information.</a></strong> 
            </p>
            <p>
                <em>Proceedings of the Eleventh International Computing and Combinatorics Conference (<strong>COCOON 2005</strong>), 2005. (Acceptance rate: around 27%.)</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Wen-Jing Hsu and Jiangdian Wang.
            </p>
            <p>
                <strong><a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/small-world.pdf">Near Optimal Routing for Small-World Networks with Augmented Local Awareness.</a></strong> 
            </p>
            <p>
                <em>Proceedings of the 3rd International Symposium on Parallel and Distributed Processing and Applications (<strong>ISPA&#39;2005</strong>), Nanjing ,China , 2-5 Nov. 2005.</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng and Wen-Jing Hsu.
            </p>
            <p>
                <strong>Off-Line AGV Routing on the 2D Mesh Topology with Partial Permutation.</strong> 
            </p>
            <p>
                <em>Proceedings of the IEEE 6th International Conference on Intelligent Transportation Systems (<strong>ITSC 2003</strong>), Shanghai, China, October 12 - 15, 2003.(<a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/itsc2003.pdf">pdf</a>) (<a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/itsc2003.ppt">ppt</a>) </em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng and Wen-Jing Hsu.
            </p>
            <p>
                <strong>Conflict-Free Routing of AGVs on the Mesh Topology Based on a Discrete-Time Model.</strong> 
            </p>
            <p>
                <em>Proceedings of 2003 IEEE International Conference on Robotics and Automation (<strong>ICRA2003</strong>), Taipei, Taiwan, Sep 14-19, 2003. (<a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/Conflict-Free%20Routing%20of%20AGVs%20on%20the%20Mesh%20Topology%20Based%20on%20a%20Discrete-Time%20Model.pdf">pdf</a>) (<a href="https://www.figma.com/exit?url=http%3A%2F%2Fgroup.iiis.tsinghua.edu.cn%2F~compbio%2Fpapers%2FConflict-Free%2520Routing%2520of%2520AGVs%2520on%2520the%2520Mesh%2520Topology%2520Based%2520on%2520a%2520Discrete-Time%2520Model.ps">ps</a>)</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Wen-Jing Hsu and Voon Yee Vee.
            </p>
            <p>
                <strong>An AGV-Routing Algorithm in the Mesh Topology with Random Partial Permutation.</strong> 
            </p>
            <p>
                <em>Proceedings of the 11th International Conference on Advanced Robotics (<strong>ICAR2003</strong>), University of Coimbra, Portugal, June 30 - July 3, 2003. (<a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/ICAR03.pdf">pdf</a>)</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Wen-Jing Hsu and Ling Qiu.
            </p>
            <p>
                <strong>An Energy-Efficient Algorithm For Conflict-Free AGV Routing On A Linear Path Layout.</strong> 
            </p>
            <p>
                <em>Proceedings of the International Computer Symposium 2002 (<strong>ICS 2002</strong>), Hualian, Taiwan, Dec. 18-21, 2002. (<a href="https://group.iiis.tsinghua.edu.cn/~compbio/papers/An%20Energy-Efficient%20Algorithm%20For%20Conflict-Free%20AGV%20Routing%20On%20A%20Linear%20Path%20Layout.pdf">pdf</a>) (<a href="https://www.figma.com/exit?url=http%3A%2F%2Fgroup.iiis.tsinghua.edu.cn%2F~compbio%2Fpapers%2FAn%2520Energy-Efficient%2520Algorithm%2520For%2520Conflict-Free%2520AGV%2520Routing%2520On%2520A%2520Linear%2520Path%2520Layout.ps">ps</a>)</em>
            </p>
        </li>
        <li>
            <p>
                Long Chen, Guodong Lu and Jianyang Zeng.
            </p>
            <p>
                <strong>Generation and Adjustment of Dimensional Models Based on Functional Semantics.</strong> 
            </p>
            <p>
                <em><strong>China Mechanical Engineering</strong>. 2007. (In Chinese).</em>
            </p>
        </li>
        <li>
            <p>
                Jianyang Zeng, Yueding Shi and Guodong Lu.
            </p>
            <p>
                <strong>An Automated Dimensioning Approach Based on the Functional Semantics.</strong> 
            </p>
            <p>
                <em><strong>China Mechanical Engineering</strong>. Vol. 14 (16), 2002. (In Chinese). </em>
            </p>
        </li>
    </ol>
</div>
</div>


<footer>
	<div class="footerBox">
		<img class='bg' src="img/footer.png" />
		<h3>CONTACT US</h3>
		<p>Address: No. 600 Dunyu Road, Sandun Town, Xihu District 310030 Hangzhou, Zhejiang PR China</p>
		<p>E-mail：zengjy@westlake.edu.cn</p>
		<a href="http://beian.miit.gov.cn/state/outPortal/loginPortal.action" target="_blank">
        <span>浙ICP备18025489号</span>
        <span>浙公安备33010602007514号</span>
        <span>Copyright © Westlake University. All Rights Reserved</span>
     </a>
	</div>
</footer>

<div id="goTop" class="goTop iconfont icon-top"></div>
<script src="js/scroll.js"></script>

</body>
</html>